bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -206,7 +200,7 @@
 {% block email_form %}{% endblock %}
 
 {% block js %}
-  {% if not l10n_has_tag('emerging_tech_links') %}
+  {% if LANG.startswith('en-') or not l10n_has_tag('emerging_tech_links') %}
```

Comment:
I see that this works, but it looks kind of backwards. If the visitor's locale does *not* have the L10n tag `emergine_tech_links`, they get the video JS? It's tough to understand why/how this tag affects including JS. Maybe add a comment to clarify?